src/GeneralStateTestsFiller/stExample/paygasRevertFiller.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
This is in `stExample`, for now, because adding a suite would need a fork of retesteth (I think.)

Reply:
:+1: